plugins/chanbackup.c

```diff
@@ -760,6 +760,128 @@ static struct command_result *json_restorefrompeer(struct command *cmd,
 				     	    NULL);
 }
 
+/* Returns True if the backup data is correct and
+ * returns false with the reason in `fail` if it's incorrect.
+ * Instead of using `decrypt_scb()` we're using this because
+ * it shuts down the node in case of any discrepancy and that
+ * is not what we want to happen when someone runs `verifybackup`. */
+static bool verify_bkp(struct command *cmd,
+					   u8 *filedata,
+					   char **fail) {
+
+	crypto_secretstream_xchacha20poly1305_state crypto_state;
```

Comment:
wow what name :)

There is any way to define a alias on top of the fine?

```
/* Re aliasing the crypto_secretstream_xchacha20poly1305_state type */
typedef crypto_secretstream_xchacha20poly1305_state bk_crypto_state;
```

Reply:
Nice idea, Thanks!